File: src/leetcode/java/two_pointers/RemoveDuplicatesFromSortedArray_26.java
package leetcode.java.two_pointers;

public class RemoveDuplicatesFromSortedArray_26 {
  public int removeDuplicates(int[] nums) {
    int i = 0;
    for (int n : nums)
      if (i < 1 || n > nums[i - 1]) nums[i++] = n;
    return i;
  }
}

File: src/leetcode/python/two_pointers/RemoveDuplicatesFromSortedArray_26.py
class RemoveDuplicatesFromSortedArray_26(object):
    def removeDuplicates(self, nums):
        i = 0
        for n in nums:
            if i < 1 or n > nums[i - 1]:
                nums[i] = n
                i += 1
        return i
